activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+*   Allow to configure maximum cache key sizes
+
+    When the key exceeds the configured limit (250 bytes by default), it will be truncated and
+    the digest of the rest of the key appended to it.
+
+    Note that previously `ActiveSupport::Cache::RedisCacheStore` allowed up to 1kb cache keys before
+    truncation, which is now reduced to 250 bytes.
```

Comment:
Ideally this new default would only apply to apps that opt in to 8.1 defaults. Then apps can carefully upgrade without inadvertently invalidating their caches.

Reply:
My thinking was that keys larger than 250 bytes are rare enough that this isn't necessary. But if you think that's not the case, yes we can introduce another framework default.